lastuser_ui/templates/edit_team.html

```diff
@@ -10,50 +10,13 @@
 
 {% block footerscripts %}
   <script type="text/javascript">
-    $("#users").select2({
-      placeholder: "Search for a user",
-      multiple: true,
-      minimumInputLength: 1,
-      ajax: {
-        url: "{{ url_for('lastuser_oauth.user_autocomplete') }}",
-        dataType: 'jsonp',
-        data: function(term, page) {
-          return {
-            q: term
-          };
-        },
-        results: function(data, page) {
-          var rdata = [];
-          if (data.status == 'ok') {
-            for (var i=0; i < data.users.length; i++) {
-              rdata.push({
-                id: data.users[i].buid, text: data.users[i].label
-              });
-            };
-          };
-          return {more: false, results: rdata};
-        }
-      },
-      initSelection: function(element, callback) {
-        var val = $(element).val();
-        if (val != '') {
-          var qs = '?userid=' + val.replace(/,/g, '&userid=');
-          $.ajax("{{ url_for('lastuser_oauth.user_get_by_userids') }}" + qs, {
-            accepts: "application/json",
-          }).done(function(data) {
-            $(element).val('');  // Clear it in preparation for incoming data
-            var rdata = [];
-            if (data.status == 'ok') {
-              for (var i=0; i < data.results.length; i++) {
-                rdata.push({
-                  id: data.results[i].buid, text: data.results[i].label
-                });
-              };
-            };
-            callback(rdata);
-          });
-        }
-      }
+    $(function(){
+      Baseframe.Forms.lastuserAutocomplete({
+        id: "users",
+        multiple: true,
+        autocomplete_endpoint: "{{ url_for('lastuser_oauth.user_autocomplete') }}",
+        getuser_endpoint: "{{ url_for('lastuser_oauth.user_get_by_userids') }}"
+      });
```

Comment:
This code should be inserted by Baseframe's `forms.html` in response to the presence of a `UserSelectField`, and the endpoint URLs should be provided to the field when defining the form or in the view.